screens/Onboarding/OnboardingScreen.tsx

```diff
@@ -24,6 +24,7 @@ const OnboardingScreen: React.FC = () => {
       {/* Username/Email Field */}
       <TextInput
         style={[styles.inputField, globalStyles.text]}
+        id="username_placeholder"
```

Comment:
Even though these are needed for the tests, they are visually polluting the code :/ unfortunately we are forced to leave it like that.